src/Util/Datasource.php

```diff
@@ -137,7 +137,7 @@ public function toMySQLArguments($tmpDir) {
     $data = "[client]\n";
     $data .= "host={$this->host}\n";
     $data .= "user={$this->username}\n";
-    $data .= "password={$this->password}\n";
+    $data .= "password='{$this->password}'\n";
```

Comment:
Good idea.

I suppose one could have a password with a quote mark?  Do you happen to know what escaping rules they use MySQL's file-format?  Maybe `addslashes()` or `addcslashes()` would come closer?